browser/ui/webui/skus_internals_ui.cc

```diff
@@ -36,4 +41,15 @@ void SkusInternalsUI::GetEventLog(GetEventLogCallback callback) {
   NOTIMPLEMENTED_LOG_ONCE();
 }
 
+void SkusInternalsUI::GetSkusState(GetSkusStateCallback callback) {
+  // TODO(simonhong): Determine which value should be displayed.
+  std::move(callback).Run("Not ready");
+}
+
+void SkusInternalsUI::ResetSkusState() {
+  VLOG(2) << __func__;
+  brave_vpn::ClearSubscriberCredential(local_state_);
```

Comment:
Reset vpn's subscriber credential here also.
W/o this, vpn panel still show main page but maybe we want to show sell panel when we do reset.